test/tests/adaptivity/scalar/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  issues = '#12538 #12309'
+  design = 'syntax/Adaptivity/index.md'
+  [./test]
+    type = 'Exodiff'
+    input = 'scalar_adaptivity.i'
+    exodiff = 'scalar_adaptivity_out.e-s002'
+    requirement = "The Adaptivity system shall support high-order SCALAR variables"
```

Comment:
```suggestion
    requirement = "The system shall support adaptivity with high-order SCALAR variables"
```

This fits the requirement standard we've chosen a bit better.

Reply:
```suggestion
    requirement = "The system shall support adaptivity with high-order scalar variables"
```
Slight tweak from @cticenhour

Reply:
Ah yeah - that should be lowercase.